README.md

```diff
@@ -14,8 +14,8 @@ the service's docker image.
 
 ```shell
 # you may need sudo for those
-apt-get install -y python-pip
```

Comment:
instead of trying to keep a list of dependencies updated in two places, we might want to tell the user to run `setup.py `, as is on container image building.